pkg/kubectl/cmd/attach.go

```diff
@@ -76,7 +77,7 @@ func NewCmdAttach(f cmdutil.Factory, cmdIn io.Reader, cmdOut, cmdErr io.Writer)
 			cmdutil.CheckErr(options.Run())
 		},
 	}
-	// TODO support UID
+	cmd.Flags().Int64Var(&options.GetPodTimeOut, "pod-running-timeout", 60, "The length of time (in seconds) to wait until at least one pod is running")
```

Comment:
Turn this into a helper, see: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/util/helpers.go